aggregate/billing_functions/gcp.py

```diff
@@ -71,7 +71,7 @@ def get_topic(row):
 
     migrate_rows = get_billing_data(start, end)
 
-    if not migrate_rows:
+    if len(migrate_rows) == 0:
```

Comment:
Why this change, I would've thought the original would be more inclusive.

Reply:
The original doesn't work since it's a dataframe. It just gives an " == meaning on a df is ambiguous". So either it's use a df specific method or check the length.